### PR TITLE
Move docker docs to another README

### PR DESCRIPTION
**Pull request by subdavis**
New README: https://github.com/VIAME/VIAME-Web/blob/fd6ac189499df3dde555a81badbf246003e4ae1e/README.md
New Docker README: https://github.com/VIAME/VIAME-Web/blob/fd6ac189499df3dde555a81badbf246003e4ae1e/docker/README.md

### CODE REVIEW

File: docker/girder.Dockerfile [hunk 1]
```diff
@@ -11,6 +12,12 @@ RUN yarn build
 
 FROM girder/girder as runtime
 
+ENV GIRDER_MONGO_URI mongodb://mongo:27017/girder
+ENV GIRDER_ADMIN_USER admin
+ENV GIRDER_ADMIN_PASSWORD viame
+ENV CELERY_BROKER_URL amqp://guest:guest@rabbit/
+ENV BROKER_CONNECTION_TIMEOUT 2
```
Comment (AlmightyYakob) [inline docker/girder.Dockerfile:19]: Why was this added? Do these serve as a default for the variables specified in `docker-compose.yml`? Or if not, how do they interfere with them?

File: docker/girder_worker.Dockerfile [hunk 2]
```diff
@@ -2,6 +2,9 @@ FROM kitware/viame:gpu-all-models-latest
 # expects VIAME install at /opt/noaa/viame/
 # expects VIAME pipelines at /opt/noaa/viame/configs/pipelines/
 
+ENV CELERY_BROKER_URL amqp://guest:guest@rabbit/
+ENV BROKER_CONNECTION_TIMEOUT 2
```
Comment (AlmightyYakob) [inline docker/girder_worker.Dockerfile:6]: Same question as above

File: docker/girder.Dockerfile (hunk 1 above)
Comment (subdavis) [reply docker/girder.Dockerfile:19]: Docker images that accept optional environment configuration should typically list them in the dockerfile.  This serves largely as documentation.  

These do serve as defaults if the images are used alone.  They're overridden by the vars supplied in docker-compose.yml
